examples/Web/appsettings.Development.json

```diff
@@ -0,0 +1,20 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+  "Kafka": {
+    "ProducerSettings": {
+      "BootstrapServers": "localhost:9092"
```

Comment:
Maybe play it cool and have a ccloud-ready config block?

Reply:
good idea.